torchmetrics/detection/map.py

```diff
@@ -160,22 +136,24 @@ class MAP(Metric):
     <https://github.com/PyTorchLightning/metrics/blob/master/tm_examples/detection_map.py>`_
 
     .. note::
-        This metric is a wrapper for the
+        This metric is following the mAP implementation of
         `pycocotools <https://github.com/cocodataset/cocoapi/tree/master/PythonAPI/pycocotools>`_,
-        which is a standard implementation for the mAP metric for object detection. Using this metric
-        therefore requires you to have `pycocotools` installed. Please install with ``pip install pycocotools`` or
-        ``pip install torchmetrics[detection]``.
+        , a standard implementation for the mAP metric for object detection.
 
     .. note::
         This metric requires you to have `torchvision` version 0.8.0 or newer installed (with corresponding
         version 1.7.0 of torch or newer). Please install with ``pip install torchvision`` or
         ``pip install torchmetrics[detection]``.
 
-    .. note::
-        As the pycocotools library cannot deal with tensors directly, all results have to be transfered
-        to the CPU, this might have an performance impact on your training.
-
     Args:
+        box_format:
+            Input format of given boxes. Supported formats are [‘xyxy’, ‘xywh’, ‘cxcywh’].
+        iou_thresholds:
+            IoU thresholds for evaluation.
+        rec_thresholds:
+            Recall thresholds for evaluation.
+        max_detection_thresholds:
+            Thresholds on max detections per image.
```

Comment:
please add more description
```suggestion
            Thresholds on max detections per image. If set to `None` will use thresholds `[1, 10, 100]`. Else please provide a list of floats.
```